srcpkgs/php/template

```diff
@@ -60,40 +59,36 @@ do_build() {
 		--enable-mysqlnd \
 		--enable-gd=shared \
 		--with-external-gd \
-		--with-zip=shared \
 		--with-ffi=shared \
 		--with-bz2=shared,$XBPS_CROSS_BASE/usr/ \
 		--with-curl=shared \
 		--with-db4=$XBPS_CROSS_BASE/usr \
 		--with-enchant=shared,$XBPS_CROSS_BASE/usr \
-		--with-freetype-dir=$XBPS_CROSS_BASE/usr \
 		--with-gdbm=$XBPS_CROSS_BASE/usr \
 		--with-gettext=shared \
 		--with-gmp=shared \
 		--with-iconv=shared \
-		--with-icu-dir=$XBPS_CROSS_BASE/usr \
-		--with-jpeg-dir=$XBPS_CROSS_BASE/usr \
 		--with-ldap=shared,$XBPS_CROSS_BASE/usr \
 		--with-ldap-sasl \
-		--with-libzip \
 		--with-sodium=shared \
```

Comment:
For these which were completely removed (also `png-dir` down below), can you document in the commit message what happened? Does PHP now always vendor them, or does it always use external now?

Reply:
sure, (most of these were noop because they would only apply if we used the vendored gd)